cc-ui/src/App.css

```diff
@@ -0,0 +1,41 @@
+#root {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
+}
+
+.logo {
```

Comment:
You can make this App.css file look cleaner by removing the parts that aren't used in the program. For example, I don't think logo was mentioned in the code anywhere